src/ray/object_manager/test/object_manager_test.cc

```diff
@@ -236,6 +238,82 @@ class TestObjectManagerCommands : public TestObjectManager {
 
   void NotificationTestComplete(ObjectID object_id_1, ObjectID object_id_2) {
     ASSERT_EQ(object_id_1, object_id_2);
+    NextWaitTest();
+  }
+
+  void NextWaitTest() {
```

Comment:
Can we separate each of the test cases into a separate unit test, and also use the gtest framework to be consistent with other raylet tests? Also, please document what case each of these is supposed to be testing.

Reply:
Yes, excellent point. I have a branch I'd like to merge before separating each test into a separate unit test as this is a somewhat substantial change.

Agree and will do the rest.